### ISSUE TITLE
the value of the `txn-total-size-limit` parameter is different in many tidb-server, and the large transactions may not execute normally. 

### ISSUE THREAD

**Issue by zhouyueyue1115** [pingcap/tidb#21435]
## Feature Request

**Is your feature request related to a problem? Please describe:**

the value of the `txn-total-size-limit` parameter is different in many tidb-server, and the large transactions may not execute normally. Also, there is no warning or check for me to find this problem.
**Describe the feature you'd like:**

after I modify the configuration or scale out a tidb-server, TiUP can check the configuration of all tidb-server, at the same time show me the differences in the configuration of all tidb-server.

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

**Comment by bbarclay** [comment]
Yes. How to get this to work in Kuberentes. We were told by a lot of people to use TiDB lighting, but this configuraiton is giving us trouble because our file size is over, for some reaons.
